lib/utilities/babel-config-for-targets.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const presetEnv = require('babel-preset-env').default;
```

Comment:
This was good advice, moved this logic to ember-cli-babel (in https://github.com/babel/ember-cli-babel/commit/78b0342f1cf0e4aae3b22dfcde10d04bce74b253).